tests/ut_main.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import database
+import EScavange
+import gamemaker
+
+class TestDatabaseCreation(unittest.TestCase):
+  #Test that we can access the database. Should not test that the database DS is correct, that should be
+  #covered by the database tests.
+  
+  def setUp(self):
+    self.db = database.Database()
+    self.gameMaker = gamemaker.GameMaker()
+  
+  def accessDB(self):
+    pass
+    #stub for now : check that we can access the database
+    # How do i verify that the database is open???
+
+class TestLogin(unittest.TestCase):
+  #Test log in functionality. Will determine if username belongs to gamemaker or team and call correct method
+  
+  def setUp(self):
+    self.db = database.Database()
+    self.inputs = EScavange.main()
```

Comment:
Make sure that this changes to escavenge.EScavenge() once name of class is changed.